### DISCUSSION TITLE
Chisel Bug

### DISCUSSION THREAD

**Discussion by JuliyJulius** [Chisel-Team/Chisel#860] [Beta Testing]
Ok, so I was actually trying to test the Chisel mod in 1.15 using Forge 1.15, but when the game opened, he showed me this image:
![image](https://user-images.githubusercontent.com/79423446/108639361-e90d7880-7472-11eb-90cf-124aba155397.png)
I don't know what this problem is supposed to mean, but there it is.
I will also send the log containing the things loaded:
https://paste.ee/p/RcD9N

**Comment by YusufSuleman** [comment]
This is meant to be in the issues tab not the discussions tab. This tab is meant for general talking about the mod.  You don't need to use PasteBin for this. As GitHub has a Code Block feature.

**Comment by tterrag1098** [comment]
1.15 is no longer supported. Please update to the latest alpha for 1.16.5
